internal/mail/backends/gomail.go

```diff
@@ -2,21 +2,29 @@ package backends
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"strconv"
 
-	"github.com/Hukyl/genesis-kma-school-entry/mail/config"
+	"github.com/Hukyl/genesis-kma-school-entry/internal/mail/config"
 	"github.com/go-gomail/gomail"
 )
 
 type GomailMailer struct {
 	config config.Config
 }
 
-func (gm *GomailMailer) SendEmail(_ context.Context, email, subject, message string) error {
+func (gm *GomailMailer) SendEmail(
```

Comment:
wait, you require a context to be passed, but don't use it? What's the point then?

Reply:
solved in next PR to avoid merge conflicts